installer.sh

```diff
@@ -119,6 +121,11 @@ while getopts cCdDFhl OPT ; do
   esac
 done
 
+# check for kali distribution
+if grep -q "kali" /etc/debian_version 2>/dev/null ; then
+    OTHER_OS=0
```

Comment:
should we also include a check to the Ubuntu version we are supporting? I think this would be more clean like
if kali
elif "check for ubuntu"
elif "check for something else in the future"
fi